app/components/JobApplication/ConfirmationView.js

```diff
@@ -14,10 +14,10 @@ const ConfirmationView = () => (
       Success! Let&#39;s match you with companies.
     </StyledParagraph>
     <p>
-      We have received your responses and you should start hearing from
-      companies soon! In the meantime, check out your{' '}
-      <LinkWrapper to="/dashboard">Dashboard</LinkWrapper> for recommended jobs,
-      and tips to build your profile. Or head over to{' '}
+      We have received your responses, and you should start hearing from
```

Comment:
Unnecessary comma preceeding an `and`

https://www.grammarly.com/blog/comma-before-and/